src/en/hentairead/src/eu/kanade/tachiyomi/extension/en/hentairead/Hentairead.kt

```diff
@@ -71,9 +74,15 @@ class Hentairead : Madara("HentaiRead", "https://hentairead.com", "en", dateForm
             listOf(
                 SChapter.create().apply {
                     name = "Chapter"
-                    url = manga.url
+                    // There's like 2 non-English entries where this breaks
+                    url = "${manga.url}english/p/1/"
```

Comment:
users would have to refresh chapter list, you can override `pageListRequest` to avoide that, altho this isn't really that important